DataFormats/PatCandidates/src/classes_def_trigger.xml

```diff
@@ -1,11 +1,15 @@
 <lcgdict>
  <selection>
 
-  <class name="pat::TriggerObject"  ClassVersion="12">
+  <class name="pat::TriggerObject"  ClassVersion="13">
+   <version ClassVersion="13" checksum="1564486686"/>
    <version ClassVersion="12" checksum="2628734905"/>
    <version ClassVersion="11" checksum="2574350865"/>
    <version ClassVersion="10" checksum="2299474032"/>
   </class>
+  <ioread sourceClass="pat::TriggerObject" version="[-12]" source="reco::CandidateBaseRef refToOrig_" targetClass="reco::CandidatePtr" target="refToOrig_">
```

Comment:
I _believe_ the `targetClass` should be
```suggestion
  <ioread sourceClass="pat::TriggerObject" version="[-12]" source="reco::CandidateBaseRef refToOrig_" targetClass="pat::TriggerObject" target="refToOrig_">
```

Reply:
Yep, and in fact without that it was not even trying to compile the CDATA, which also had a typo.